src/controllers/authController.js

```diff
@@ -54,3 +54,51 @@ exports.registerNewUser = (req, res) => {
 		}
 	);
 };
+
+exports.login = async (req, res) => {
+	try {
+		const prevUser = await user.findOne({ username: req.body.username });
```

Comment:
`user` here is undefined. The correct model name is `User` as seen in line 1
Also, the login should be with email not username